client/src/components/Tag/CompactTagList.js

```diff
@@ -9,41 +9,21 @@ import SummaryTag from './SummaryTag';
  * Extension of TagList that is aware of its size and switch to a summary view if greater than a
  * specific width.
  */
-class CompactTagList extends Component {
-  constructor(props) {
-    super(props);
-
-    this.render = this.render.bind(this);
-    this.onResize = this.onResize.bind(this);
-    this.refreshShowSummaryView = this.refreshShowSummaryView.bind(this);
-    this.getPlaceholderSize = this.getPlaceholderSize.bind(this);
-
-    this.state = {
-      showSummaryView: false
-    };
-  }
-
-  componentDidUpdate() {
-    const placeholderWidth = this.getPlaceholderSize();
-    this.refreshShowSummaryView(placeholderWidth);
-  }
+const CompactTagList = ({
+  maxSize = 0,
+  onSummary = () => {},
+  ...listProps
```

Comment:
Previously listProps was defined as const { maxSize, onSummary, ...listProps } = this.props; so this is correct